lib/EncodeImagesStage.js

```diff
@@ -4,20 +4,32 @@ var Promise = require('bluebird');
 
 Jimp.prototype.getBufferAsync = Promise.promisify(Jimp.prototype.getBuffer);
 
-module.exports = encodeImages;
+module.exports = EncodeImagesStage;
+
+/**
+ * Encodes Jimp images in pipeline extras to buffers.
```

Comment:
Also I would not duplicate the doc for `run` here.  This just constructs the object for the stage; it can be very concise and just link to the `run` function.

Please build and verify the doc locally as well.